language/tools/move-cli/src/package/cli.rs

```diff
@@ -496,57 +489,38 @@ pub fn run_move_unit_tests(
                 .collect::<HashMap<_, _>>()
         })
         .collect();
+    let root_package = Some(resolution_graph.root_package.package.name);
     let build_plan = BuildPlan::create(resolution_graph)?;
     // Compile the package now. We need to treat the root package differently since we need to
```

Comment:
I think that this comment is largely out of date since the code does not specialize towards the root package anymore.